### DISCUSSION TITLE
送绝区零100级账号

### DISCUSSION THREAD

**Discussion by yuanzidamn** [share121/inter-knot#224] [其他]
把你父母的银行卡密码给我

**Comment by yuanzidamn** [comment]
如果你点进来了，不要相信，治安局提醒大家免费赠送勿轻信，谨慎扫码莫贪心

**Comment by yiwangzj** [comment]
最近诈骗频发，绳网也没有逃过，我今天还看到绳网上一个诈骗求助贴，说要集钱治疗他的长毛兄弟，大家一定要小心不要上当！

**Reply by dongzhiaidao** [reply]
兄弟生病了，急需328在20号之前急救！（268也行）.JPG

**Comment by Mu-7593** [comment]
[[新艾利都治安局提醒您：绳网不是法外之地，为了保护您和他人的人身及财产安全，请勿传播虚假不实信息]](https://www.bilibili.com/video/BV1im42137GG/?share_source=copy_web&vd_source=7c3c8a1f75ec9390ef01002167beab02)

**Comment by dongzhiaidao** [comment]
兄弟生病了，急需328在20号之前急救！（268也行）.JPG

**Comment by Kamisato100** [comment]
你信天上会掉免费的菲林，还是相信我是秦始皇派蒙
